main.js

```diff
@@ -11,93 +10,15 @@ class Book {
   }
 }
 
-class Store {
-  getBooks = () => {
-    let books;
-    if (localStorage.getItem('booksStored') === null) {
-      books = [];
-    } else {
-      books = JSON.parse(window.localStorage.getItem('booksStored'));
-    }
-    return books;
-  }
-
-  addBook = (book) => {
-    const books = this.getBooks();
-
-    books.push(book);
-    window.localStorage.setItem('booksStored', JSON.stringify(books));
-  }
-
-  removeBook = (button) => {
-    const books = this.getBooks();
-    const parentDiv = button.parentNode;
-    const myTitle = parentDiv.querySelector('.title').textContent;
-    const myAuthor = parentDiv.querySelector('.author').textContent;
-    const booksLeft = books.filter((book) => book.title !== myTitle && book.author !== myAuthor);
-
-    window.localStorage.setItem('booksStored', JSON.stringify(booksLeft));
-  }
-}
-
-class MainUI {
-  populateBooks = () => {
-    const store = new Store();
-    const books = store.getBooks();
-    books.forEach((book) => {
-      this.createBook(book);
-    });
-  }
-
-  createBook = (book) => {
-    const booksContainer = document.querySelector('.books-container');
-    const bookContainer = document.createElement('div');
-    bookContainer.className = 'single-book';
-    const authorContainer = document.createElement('div');
-    authorContainer.className = 'author-name';
-    const span = document.createElement('span');
-    span.textContent = 'by';
-    const bookTitle = document.createElement('h3');
-    bookTitle.className = 'title';
-    const bookAuthor = document.createElement('h3');
-    bookAuthor.className = 'author';
-    const removeButton = document.createElement('button');
-    removeButton.className = 'remove-button';
-    removeButton.textContent = 'Remove';
-    bookTitle.textContent = book.title;
-    bookAuthor.textContent = book.author;
-    authorContainer.append(bookTitle, span, bookAuthor);
-    bookContainer.append(authorContainer, removeButton);
-
-    booksContainer.appendChild(bookContainer);
-  }
-
-  addBook = (book) => {
-    this.createBook(book);
-  }
-
-  removeBook = (button) => {
-    const parentDiv = button.parentNode;
-    parentDiv.remove();
-  }
-
-  clearInputs = () => {
-    form.elements.title.value = '';
-    form.elements.author.value = '';
-  }
-}
-
 const mainPage = new MainUI();
-const store = new Store();
-
 const addBookSection = document.querySelector('.bottom-container');
 const contactSection = document.querySelector('.contact');
 const displayBookSection = document.querySelector('.top-container');
-
 // Date
 const date = document.querySelector('.date');
-const dateTime = luxon.DateTime.utc().toLocaleString(luxon.DateTime.DATETIME_FULL);
-date.textContent = dateTime;
+setInterval(() => {
+  date.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+}, 1000);
```

Comment:
- [ ] Kindly add the current date below the navigation bar, to match the requirements.

![image](https://user-images.githubusercontent.com/35535130/159439842-4cdc3557-06b1-4cf6-a420-a095796615b9.png)

Reply:
resolved 👍